src/render.cc

```diff
@@ -4,6 +4,9 @@
 #include <GL/gl3w.h> // Runtime function loader (core only)
 #include <GLFW/glfw3.h> // OpenGL included
 #include "render.h"
+#include <unistd.h>
+
+#define MIN(a, b) ((a > b) ? b : a)
```

Comment:
I might not use this macro, so it probably should be removed since it's such a common name that can conflict with other library symbol names.